README.rst

```diff
@@ -37,174 +37,33 @@ Run your jobs:
 
 ::
 
-    studio run myfile.py
+    studio run train_mnist.py
```

Comment:
I have spent much more time polishing train_mnist_keras.py, let's put that one here